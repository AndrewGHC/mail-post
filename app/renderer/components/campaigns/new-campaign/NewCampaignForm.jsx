import React from 'react';
import PropTypes from 'prop-types';
import Paper from '@material-ui/core/Paper';
import { withStyles } from '@material-ui/core/styles';

import NewCampaignFormName from './NewCampaignFormName';

const styles = theme => ({
  root: theme.mixins.gutters({
    paddingTop: 16,
    paddingBottom: 16,
    marginTop: theme.spacing.unit * 3,
  }),
});

const NewCampaign = ({ classes }) => (
  <Paper className={classes.root}>
    <NewCampaignFormName />
  </Paper>
);

NewCampaign.propTypes = {
  classes: PropTypes.object.isRequired, // eslint-disable-line react/forbid-prop-types
};

export default withStyles(styles)(NewCampaign);
