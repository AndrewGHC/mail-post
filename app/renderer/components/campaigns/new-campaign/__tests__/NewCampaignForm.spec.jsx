import React from 'react';
import { shallow } from 'enzyme';

import NewCampaignForm from '../NewCampaignForm';

describe('NewCampaignForm', () => {
  it('matches snapshot', async () => {
    const wrapper = shallow(<NewCampaignForm />);
    expect(wrapper).toMatchSnapshot();
  });
});
