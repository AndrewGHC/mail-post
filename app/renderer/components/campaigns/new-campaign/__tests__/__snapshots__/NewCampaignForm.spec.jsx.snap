// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewCampaignForm matches snapshot 1`] = `
<NewCampaign
  classes={
    Object {
      "root": "NewCampaign-root-1",
    }
  }
/>
`;
